carma-messenger-ui/website/widgets/truckInspection/widget.js

```diff
@@ -313,6 +318,22 @@ CarmaJS.WidgetFramework.truckInspection = (function () {
 
             console.log("truckInspectionSafetyLogPerVIN element is created");
     }
+
+    var IsValidVINNumber = function(vin_number)
+    {
+        let isValid = false;
+        let valid_truck_vin_numbers = CarmaJS.Config.getValidTruckVINNumber();
+        valid_truck_vin_numbers.forEach((valid_vin)=>{
+            console.log("Valid VIN: " + valid_vin);
+            console.log("Check VIN: " + vin_number);
+            if(valid_vin.trim() == vin_number.trim()){
+                console.log("Found a valid VIN");
+                isValid = true;
+            }
```

Comment:
@dan-du-car please put an else statement to show "invalid vin found"

Reply:
I would still prefer to have console log saying: "Found valid VIN #####" or Found invalid VIN #####. especially if there's only one item we're iterating in the forEach((valid_vin)) loop.